README.md

```diff
@@ -26,8 +26,14 @@ This is how a `package.json` could look like when making use of these scripts:
   "devDependencies": {
     "uphold-scripts": "git+ssh://git@github.com/uphold/uphold-scripts#v0.1.0"
   },
+  "lint-staged": {
+    "linters": {
```

Comment:
@cristianooliveira, @rplopes I've updated the PR with the suggested changes. I also upgraded lint-staged to the latest release. Here's the [list of commits](https://github.com/okonet/lint-staged/compare/v9.2.0...v9.5.0) since the version we had.